README.md

```diff
@@ -1 +1,22 @@
-# api-server
\ No newline at end of file
+# api-server
+
+LAB - Class 4
+Project: api server
+Author: garrett
+Links and Resources
+
+
+
+
+PORT - 3000
+MONGODB_URI - there should be one bu its not working its probably not even made yet idk
+How to initialize/run your application (where applicable)
+ npm start
+
+Tests
+no tests yet
+
+Describe any tests that you did not complete, skipped, etc
+all of them
+UML
+i dont know what a uml is
```

Comment:
Here is a website that can help with UML:

https://developer.ibm.com/articles/an-introduction-to-uml/